docs/user-manual/en/configuration-index.md

```diff
@@ -210,6 +210,7 @@ name | node name; used in topology notifications if set. | n/a
 [read-whole-page](paging.md) | If true the whole page would be read, otherwise just seek and read while getting message. | `false`
 [paging-directory](paging.md#configuration)| the directory to store paged messages in. | `data/paging`
 [persist-delivery-count-before-delivery](undelivered-messages.md#delivery-count-persistence) | True means that the delivery count is persisted before delivery. False means that this only happens after a message has been cancelled. | `false`
+[persist-redelivery](undelivered-messages.md#persist-redelivery) | True means that a journal update record is added every time a redelivery occurs. If your system is playing with short redeliveries this should be set to false (recommended value). | `true`
```

Comment:
The term 'short redeliveries' doesn't really read well to me, seems more accurate to write it as 'short redelivery delays' based on what I think you mean.

Reply:
@tabish121 I just simplified the statement.

I was going to complete removal as I don't see much value on keeping these records at all. (I don't think it's a big deal to always delivery the message right away or to reset the delivery count once the broker is restarted).. but I may get users to disagree with me.

if you could help me with a better wording?